python/sglang/srt/lora/layers.py

```diff
@@ -43,7 +49,24 @@ def slice_lora_b_weights(self, B: torch.Tensor, tp_rank: int):
         pass
 
 
-class VocabParallelEmbeddingWithLoRA(BaseLayerWithLoRA):
+class EmbeddingLayerWithLoRA(BaseLayerWithLoRA):
```

Comment:
What's the purpose of this class? If we are not relying on it for polymorphism, maybe we can have the two subclassess to base on BaseLayerWithLoRA directly?